docs/conf.py

```diff
@@ -9,7 +9,7 @@
 source_suffix = '.rst'
 master_doc = 'index'
 project = 'Mailu'
-copyright = '2018, Mailu authors'
+copyright = '2023, Mailu authors'
```

Comment:
Copyright doesn't work that way. We should provide the first year "Mailu" was used (which may be earlier than 2018). We can have both if you want.

```20XX-2023, ...```

Reply:
done in 63ab4938c94ebd14a985e096838af19385f9d966